### PR TITLE
Update links to pydmodbus-dev

### PR DESCRIPTION
**Pull request by alexrudd2**
- Rename all links `riptideio`->`pymodbus-dev`.  This fixes the install issue @aritakku was facing

> Tried again, and the installation via instructions does not work:
> `git clone git://github.com/riptideio/pymodbus.git`

_Originally posted by @aritakku in https://github.com/pymodbus-dev/pymodbus/discussions/1290#discussioncomment-4766501_

- Add HTML lang attribute to satisfy HTML linters.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -91,8 +91,6 @@ base operating system will allow (*allow* in this case means how many Virtual IP
 
 For more information please browse the project documentation:
 
-https://riptideio.github.io/pymodbus/
```
Comment (alexrudd2) [inline README.rst:94]: I couldn't find the GitHub pages for the new name, so I just deleted this link

Comment (janiversen) [reply README.rst:94]: yes this should be readthedocs, which is our official place for documentation.

hmmm, I might actually have missed that one.

**Comment by janiversen** [comment]
Seems you have a lot of good changes here! let me just do the review.

**Review by janiversen — APPROVED** [review]
Good catches!

you have done with lang=en, where my PR was only riptideio --> pymodbus-dev

File: README.rst [hunk 2]
```diff
@@ -5,8 +5,7 @@ We are happy to announce that we have a new home: pymodbus-dev, which is pure 10
 The move from a company organization to pymodbus-dev was done to allow a 100% openness in the spirit of FOSS.
 
 .. image:: https://github.com/pymodbus-dev/pymodbus/actions/workflows/ci.yml/badge.svg?branch=dev
-   :target: https://github.com/pymodbus-dev/pymodbus/actions/workflows/ci.yml
-.. image:: https://readthedocs.org/projects/pymodbus/badge/?version=latest
+   :target: https://github.com/pymodbus-dev/pymodbus/actions/workflows/ci.yml.. image:: https://readthedocs.org/projects/pymodbus/badge/?version=latest
```
Comment (janiversen) [inline README.rst:8]:
```suggestion
   :target: https://github.com/pymodbus-dev/pymodbus/actions/workflows/ci.yml
 .. image:: https://readthedocs.org/projects/pymodbus/badge/?version=latest
```

**Review by janiversen — CHANGES_REQUESTED** [review]

**Review by janiversen — APPROVED** [review]

**Comment by janiversen** [comment]
Thanks.

**Pull request merged**